CHANGELOG.md

```diff
@@ -6,9 +6,13 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 The crates in this repository do not adhere to [Semantic Versioning](https://semver.org/spec/v2.0.0.html) at this time.
 
-## Unreleased
+## [4.0.1]
 
-- Improved performance of mobilecoind-dev-faucet for better load testing
+### Added/Changed
+
+- Improved performance of mobilecoind-dev-faucet for better load testing ([#2955])
+- Make the local network script support a dense3 network option ([#2988])
+- HAProxy Ingress Blocklists ([#2952])
```

Comment:
:upside_down_face: Suggest separate sections for "Added" and "Changed" 
```suggestion
## [4.0.1]

### Added

- Make the local network script support a dense3 network option ([#2988])
- HAProxy Ingress Blocklists ([#2952])

### Changed

- Improved performance of mobilecoind-dev-faucet for better load testing ([#2955])
```

Reply:
This should get applied.